monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -150,13 +196,87 @@ class ConfigurePageComponent extends AuthComponent {
       });
   };
 
+  PBAwindows = () => {
+    return (<FilePond
+      server={{ url:'/api/fileUpload/PBAwindows',
+                process: {headers: {'Authorization': this.jwtHeader}},
```

Comment:
We need to somehow change all these headers when JWT changes, but I spent a lot of time and couldn't do it :/
Left it as is, because it's unlikely that user will stay for >5 minutes on monkey configuration and only then upload files. Bug **doesn't** happen with auth turned ON.